components/log_wrappers/Cargo.toml

```diff
@@ -6,6 +6,17 @@ publish = false
 
 [dependencies]
 hex = "0.4"
+kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+protobuf = "2.8"
 slog = "2.3"
 slog-term = "2.4"
 tikv_alloc = { path = "../tikv_alloc" }
+paste = "1.0"
```

Comment:
It's unnecessary anymore.

Reply:
I've removed this dependency. By the way, `hex` crate is not needed for most of the crates. I've removed them altogether.